### ISSUE TITLE
Cannot use 'in' operator to search for 'go' in undefined

### ISSUE THREAD

**Issue by frozonfreak** [NorthwoodsSoftware/GoJS#70]
While compiling production build for my project following error occurs 

```
Uncaught TypeError: Cannot use 'in' operator to search for 'go' in undefined
    at go.js:2078
    at Object.<anonymous> (go.js:8)
    at Object.6f70 (go.js:8)
    at l (bootstrap:88)
    at Module.56d7 (App.vue?bff9:21)
    at l (bootstrap:88)
    at Object.1 (_base-input.vue?a136:29)
    at l (bootstrap:88)
    at a (bootstrap:45)
```

Error happens only in build 2.0.0. Changing revision to 1.8.36 solves the problem.

**Comment by simonsarris** [comment]
Thanks for reporting this. What are you using to compile? What is your project built on.

**Comment by simonsarris** [comment]
This should be fixed in 2.0.2. Please let us know if there's still a problem.
